warehouse/templates/accounts/organization-invite-confirmation.html

```diff
@@ -17,8 +17,16 @@
     {% trans %}Confirm Organization Invite{% endtrans %}
 {% endblock %}
 
-{% block main %}
+{% block prompt %}
+  {% if request.user.has_two_factor %}
+    {{ super() }}
+  {% else %}
+    <!-- Content hidden -->
+  {% endif %}
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
**praise:** ‏I hadn't seen this pattern used, nice way to hide the confirmation.